KLAYTN_README.md

```diff
@@ -0,0 +1,89 @@
+# Klaytn API Options and Evaluation
+
+We are currently evaluating two options to retrieve Klaytn NFT metadata. These examples and evaluation will shed light on the options.
+
+## Klaytn API
+
+### Resources
+
+- [DEV Documentation](https://www.klaytn.com/developers)
+- [Klaytn Documentation](https://docs.klaytn.com)
+- [API keys](https://dashboard.blockchainapi.com/#api-keys) (check 1password)
+- [SDK w/caver-js](https://medium.com/klaytn/common-architecture-of-caver-a714224a0047)
+- [Klaystagram Tutorial](https://docs.klaytn.com/dapp/tutorials/klaystagram)
+
+### Setup and Run
+
+You will need to get KLAY into a wallet, which you can obtain at: https://docs.klaytn.com/dapp/developer-tools/klaytn-wallet#how-to-receive-baobab-testnet-klay. 
+
+$ npm install caver-js
+
+An API pair is required for The Blockchain API and Klaytn. 
```

Comment:
Is this in 1Password? If so, noting that would be helpful.

Reply:
Yup, in 1Pass under Developers>API Key